pkg/broker/ingress/stats_reporter.go

```diff
@@ -33,15 +33,15 @@ var (
 	// eventCountM is a counter which records the number of events received
 	// by the Broker.
 	eventCountM = stats.Int64(
-		"event_count",
+		"broker_ingress_event_count",
```

Comment:
same thing here... why do we need this change? is for a UT coverage thing?
we are matching the naming convention of metrics in serving..

Reply:
Code coverage can not run with the two init methods registering the two matching names. Plus this prevents us from running in the same container...

Reply:
code coverage is not a blocker for merging things. If there is a workaround without changing the names, then go for it... Otherwise we should not change the names.

When the time comes to run in the same container, then I think we can revisit.